### DISCUSSION TITLE
Organizing Prompt Templates in LangSmith Web UI

### DISCUSSION THREAD

**Discussion by dennisofficial** [langchain-ai/langsmith-sdk#943] [Ideas]
### Feature request

As the title states, it would be GREAT if we can organize our prompts in the langsmith hub. Our project uses alot of templates, and multiple templates for one chain, and it would be great to keep them organized, instead of trying to go through a long list.

### Motivation

It is frustrating to scroll through a bunch of prompts, or trying to figure out which prompt is for what chain, would be nice to organize based on chain, or however we would like to structure it.

**Comment by hinthornw** [comment]
We're starting to roll out [resource tags](https://docs.smith.langchain.com/how_to_guides/setup/set_up_resource_tags) to help flexibly organize various resources by application/environment/etc.


Hopefully it's a start
